src/platform/imx8/platform.c

```diff
@@ -28,6 +28,9 @@
 #include <errno.h>
 #include <stdint.h>
 
+#define I_CACHE_ATTRIBUTE	0x2524224F
+#define D_CACHE_ATTRIBUTE	0x2521224F
+
```

Comment:
Please explain in the commit message why we choose this exact value.

Reply:
Updated.